### ISSUE TITLE
Uploaded .pdf s still making it to the system?

### ISSUE THREAD

**Issue by harryharrold** [NeonOpenAG/oagtoolbox#191]
See attached - this shouldn't be happening any more.

![screenshot_20171125-085631](https://user-images.githubusercontent.com/907596/33337088-c3f9e8f0-d469-11e7-8433-8c879e7df340.png)

**Comment by tobybatch** [comment]
I think the current file name is still being set.

To replicate, clear all docs, upload a pdf and see it appear as current active doc.

**Comment by jermyk** [comment]
![screenshot_012618_105348_am](https://user-images.githubusercontent.com/4446325/35436663-30df9474-0287-11e8-860a-e7b5cdf31196.jpg)

**Comment by jermyk** [comment]
![screenshot_012618_111851_am](https://user-images.githubusercontent.com/4446325/35437528-b07571d8-028a-11e8-9246-a2c457d0ccc6.jpg)

**Comment by harryharrold** [comment]
Yeah - that .pdf is still making it on to the system - you;ll see it on the dev. instance until you upload an xml file. 

@tobybatch - this blocks release, please fix.
